backend/api/Program.cs

```diff
@@ -27,6 +27,7 @@ public static void Main(string[] args)
         public static IWebHostBuilder CreateWebHostBuilder(string[] args) =>
             WebHost.CreateDefaultBuilder(args)
             .UseContentRoot(Directory.GetCurrentDirectory())
+            .UseUrls("http://*:8080")
```

Comment:
This forces the app to run on 8080. Not strictly required, but does allow me to apply one healthcheck config to both local and openshift.

Reply:
See my previous comment. By using an env var we could remove this line. Thoughts?